### DISCUSSION TITLE
How to reduce the distance between the icon and the border?

### DISCUSSION THREAD

**Discussion by ffs-akl** [LBCrion/sfwbar#246] [Q&A] [answered]
If I want to adjust this distance to 0px, where should I adjust it?  
![swappy-20241016-170448](https://github.com/user-attachments/assets/96fa4736-002a-4091-99a6-c484938dcd38)

**Comment by LBCrion** [comment, marked as answer]
Set padding, margin and border for the taskbaritem image nodes to 0px; and
-GtkWidget-valign: start; to avoid vertically centering the image in the
available space.

**Reply by ffs-akl** [reply]
I added the following to sfwbar.config, but it didn't work.
```
#taskbaritem image {
  padding: 0px;
  margin: 0px;
  -GtkWidget-valign: start;
}
```

**Comment by LBCrion** [comment]
#taskbar_item image - need an underscore

**Reply by ffs-akl** [reply]
Still haven't seen any changes.

**Comment by LBCrion** [comment]
Do you by any chance have any other sections overriding the style? I.e.
other #taskbar_item or #taskbar_normal, #taskbar_active image etc sections?

**Reply by ffs-akl** [reply]
indeed, the padding of takbar_normal image overrided the style.  thank you!
